.github/workflows/deploy.yml

```
@@ -32,6 +32,13 @@ jobs:
         with:
           useConfigFile: true
 
+      - name: Create github.secrets.php
+        run: |
```

actionlint

shellcheck reported issue in this script: SC1009:info:4:1: The mentioned syntax error was in this simple command

(shellcheck)